app/uk/gov/hmrc/exports/services/SubmissionService.scala

```diff
@@ -343,4 +344,17 @@ class SubmissionService @Inject() (
       wcoPointers
     }
   }
+
+  private def pointersAfterRoutingCountriesToParent(fieldPointers: Seq[String], decId: String): Seq[String] = {
+
+    val routingCountriesPredicate = "declaration.locations.routingCountries"
+    val consignmentPointer = "declaration.consignment"
+
+    val (matches, unmatches) = fieldPointers.partition(_.contains(routingCountriesPredicate))
+
+    if (matches.nonEmpty) {
+      logProgress(decId, s"Converted ${matches.length} Routing Country pointers to 1 Consignment pointer")
+      unmatches :+ consignmentPointer
+    } else fieldPointers
+  }
```

Comment:
This is not needed. All we need to do is modify the `locations.routingCountries.$1"` entry in the mapping file to point to the WCO pointer for consignment